src/Data/Bits/Extras.hs

```diff
@@ -20,6 +20,11 @@
 module Data.Bits.Extras
   ( Ranked(..)
   , log2
+#if __GLASGOW_HASKELL__ >= 900
+  , integerLog2
+  , wordLog2
+  , word32Log2
```

Comment:
Move `word32Log2` out of this CPP block, as it's defined on all versions of GHC.